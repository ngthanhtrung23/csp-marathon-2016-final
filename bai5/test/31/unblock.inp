1.2333
1.24..
1004.5
6677.5
8899.5
......
