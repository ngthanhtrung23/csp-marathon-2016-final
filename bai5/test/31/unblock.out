32
5 U 1
7 R 1
6 R 1
9 R 2
8 R 1
1 D 3
0 L 1
2 D 1
3 L 1
5 U 1
7 R 1
4 D 3
7 L 1
5 D 1
3 R 1
2 U 1
0 R 3
1 U 3
6 L 1
8 L 1
2 D 4
0 L 2
6 R 1
1 D 1
3 L 3
5 U 1
7 R 1
4 U 4
7 L 1
9 L 1
5 D 3
0 R 3

